exercises/Laisha-Calalpa/short-response.md

```diff
@@ -0,0 +1,17 @@
+# Unit 2 - Lesson 3: Introduction to JavaScript Functions
+## Short Response Questions
+
+1. What is the difference between function parameters and arguments?
+    Parameters are variables that a function uses as a part of its function declaration.
+    Arguments are what a function takes in when the function is invoked. 
+
+2. What happens when you invoke a JavaScript function with a missing argument? What about calling a function with _extra_ arguments? 
+    The function will execute to undefined if it has missing arguments. Since there is no value being given to the function there is nothing to undergo the operations in the function. An extra argument will be discarded as if it were not there. 
```

Comment:
>  The function will ~~execute to~~ **pass in any missing arguments as**  undefined if it has missing arguments.

> Since there is no value being given to the function there is nothing to undergo the operations in the function.

Not correct, since operations can still be performed on `undefined`.

> An extra argument will be ~~discarded~~ **ignored** as if it were not there.